static/sass/styles.scss

```diff
@@ -60,6 +60,9 @@ $color-accent: $color-brand;
 @import 'patterns_sticky_footer';
 @include p-sticky-footer;
 
+@import '_snapcraft_navigation';
+@include snapcraft-navigation;
```

Comment:
The mixin should follow the Vanilla naming convention `snapcraft-p-navigation`.

Reply:
done